src/Symfony/Component/Cache/Tests/Adapter/FilesystemTest.php

```diff
@@ -27,4 +27,28 @@ public function createCachePool()
 
         return new FilesystemAdapter('sf-cache');
     }
+
+    public static function tearDownAfterClass()
+    {
+        self::rmdir(sys_get_temp_dir().'/symfony-cache');
+    }
+
+    public static function rmdir($dir)
+    {
+        if (!$dir || 0 !== strpos(dirname($dir), sys_get_temp_dir())) {
+            throw new \Exception(__METHOD__."() operates only on subdirs of system's temp dir");
+        }
+        $children = new \RecursiveIteratorIterator(
+            new \RecursiveDirectoryIterator($dir, \RecursiveDirectoryIterator::SKIP_DOTS),
+            \RecursiveIteratorIterator::CHILD_FIRST
+        );
+        foreach($children as $child) {
```

Comment:
Thank you @nicolas-grekas.
@fabpot, could you please tell me which version of `fabpot/php-cs-fixer` is using fabbot.io? Indeed, IMO it would be nice if fabbot.io can inform directly which version is running. Thank you in advance.